src/Compiler/Driver/GraphChecking/Docs.md

```diff
@@ -0,0 +1,223 @@
+##  Parallel type-checking in F﻿Sharp
```

Comment:
We should review this docs file. It's probably fairly outdated - we should update it.

Reply:
Also maybe add a link to this from the main docs.